stix_shifter_modules/darktrace/configuration/lang_en.json

```diff
@@ -2,7 +2,7 @@
     "connection": {
         "host": {
             "label": "Management IP address or Hostname",
-            "description": "Specify the IP address or hostname of the data source so that IBM Cloud Pak for Security can communicate with it"
+            "description": "Specify the IP address or hostname of the data source"
```

Comment:
is there any specific commit causing this change? not sure why this description has changed in some connectors

Reply:
same as above